content/en/continuous_testing/_index.md

```diff
@@ -39,19 +39,21 @@ Datadog Continuous Testing offers a set of tools that enable you to automate sof
 
 ## Test with ease and speed
 
-Use scalable features such as a codeless [web recorder][2], [mobile app recorder][15], [parallel test runs][3], and built-in multi-location testing to save time and effort for your QA team.
+Use scalable features such as a codeless [web recorder][2], [mobile app recorder][15], [parallel test runs][3], and built-in multi-location testing to save time and effort for your QA team. You can run your tests sequentially and customize the number of tests you want to run at the same time on the [**Settings** page][3].
```

Comment:
Is the [3] link to parallel test runs correct? I'm surprised it goes to the same place as the Settings page.

Reply:
Yes, it is. Users set their test parallelization limits on the [Settings page](https://docs.datadoghq.com/continuous_testing/settings/#parallelization).